main/views.py

```diff
@@ -20,3 +24,25 @@ def privacy(request: HttpRequest) -> HttpResponse:
       request: A GET request.
     """
     return render(request=request, template_name="main/privacy.html")
+
+
+def demo(request: HttpRequest) -> HttpResponse:
+    """View that renders a demo of the logged in app.
+
+    Args:
+      request: A GET request.
+    """
+
+    # TODO: Get this from database instead
+    def generateRandomData(n: int, categoryCount: int) -> list[dict[str, str | int]]:
+        return [
+            {
+                "skill": f"skill-{i}",
+                "category": f"category-{floor(random() * categoryCount)}",
+                "lvl": floor(random() * 10),
```

Comment:
The name for the database table is `SkillLevel`. It's probably better to change this to key to "skilllevel" to make it easier to convert.

Also, the [skill levels range from 0 - 5](https://github.com/direct-framework/digital-research-competencies-framework/blob/main/skill-levels.md), so if your visualisation framework is assuming a max value, go with 5.